src/app/components/hooks/useRegistration.tsx

```diff
@@ -59,46 +58,18 @@ const useRegistration = ({ isTeacher }: RegistrationOptions) => {
 
     const newUser = { ...registrationUser, loggedIn: false };
 
-    const handleTeacherRegistration = () => {
+    const handleTeacherRegistration = async () => {
```

Comment:
How come this is now asynchronous, but then in the only place where you're calling it you don't do `await`? Doesn't that mean that it does not need to be asynchronous?

Reply:
I don't believe that's required here; I'm not waiting for handleTeacherRegistration to execute before continuing executing more code.  The async/await within the function is required because if I get rid of those then it tries to submit the teacher request form before the account is actually created.